src/main/scala/mesosphere/marathon/event/http/HttpEventActor.scala

```diff
@@ -96,7 +96,13 @@ class HttpEventActor(conf: HttpEventConfiguration,
     //remove all unsubscribed callback listener
     limiter = limiter.filterKeys(subscribers.urls).iterator.toMap.withDefaultValue(NoLimit)
     metrics.skippedCallbacks.mark(limited.size)
-    active.foreach(post(_, event, self))
+    active.foreach(url => Try(post(url, event, self)) match {
```

Comment:
> [If the given string is not a valid URI the method throws an `IllegalUriException`.](https://github.com/spray/spray/blob/master/spray-http/src/main/scala/spray/http/Uri.scala#L210)

After merging this PR it will be impossible to set invalid callback but it could be set before and I want to prevent this old entry to stop other events.